File: generate_pins.js
const fs = require('fs');

const generatePins = () => {
    const pins = [];
    for (let i = 0; i < 1000; i++) {
        const serialNumber = 'GFA-' + (10000 + i).toString();
        // Generate a random 6-digit pin
        const pin = Math.floor(100000 + Math.random() * 900000).toString();
        pins.push({
            serialNumber,
            pin,
            used: false,
            data: null
        });
    }
    
    const fileContent = `const GFA_PINS = ${JSON.stringify(pins, null, 2)};`;
    fs.writeFileSync('data.js', fileContent);
    console.log('Successfully generated data.js with 1000 pins.');
};

generatePins();

File: generate_pins.py
import json
import random

pins = []
for i in range(1000):
    serialNumber = f"GFA-{10000 + i}"
    pin = str(random.randint(100000, 999999))
    pins.append({
        "serialNumber": serialNumber,
        "pin": pin
    })

file_content = f"const GFA_PINS = {json.dumps(pins, indent=2)};"

with open("data.js", "w") as f:
    f.write(file_content)

print("Created data.js with 1000 pins.")
